docs/index.rst

```diff
@@ -7,7 +7,8 @@
 Welcome to pip-tools' documentation!
 ====================================
 
-.. include:: ../README.rst
+.. include:: ../README.md
+   :parser: markdown
```

Comment:
Apparently not... I tried that out, but it didn't work even when I adjusted the [configuration](https://www.sphinx-doc.org/en/master/usage/markdown.html#configuration).

> Also, did you compare the rendering of the current RTD front page vs. this PR?

They differ a little, the size of the captions is not exactly the same, and the new version has a table of contents for each page. But I think it's all right (you can also check the build result yourself one last time).

Reply:
Ah, maybe that setting doesn't apply to includes..

Reply:
As for the title sizes, I think MyST has an option to offset that.

Reply:
There are some more problems, as I realized while scanning through the RTD build.

Regarding the badges and picture at the beginning of the README, the reference-style links don't render correctly.
Furthermore, syntax and console highlighting seem broken in some places. Also, the version table a the end is broken.

I do not know why all this renders correctly locally, but not here.

Reply:
Added a comment on the relevant line.

Reply:
> Also, the version table at the end is broken.

That would have to be fixed before merging. Tables can be tricky (both in MD and RST). I'd start by copying a table from elsewhere and adopting it. You could also try removing some lines to see if that makes it work — this would mean that only some lines might be broken and not the entire thing.